main.go

```diff
@@ -85,3 +82,18 @@ func open(name string, cli *http.Client) (io.ReadCloser, error) {
 	}
 	return resp.Body, err
 }
+
+// httpClient returns http client with default options
+func httpClient(insecure bool) *http.Client {
+	defaultTransport := http.DefaultTransport.(*http.Transport)
+	transport := &http.Transport{
+		Proxy:                 defaultTransport.Proxy,
+		DialContext:           defaultTransport.DialContext,
+		MaxIdleConns:          defaultTransport.MaxIdleConns,
+		IdleConnTimeout:       defaultTransport.IdleConnTimeout,
+		ExpectContinueTimeout: defaultTransport.ExpectContinueTimeout,
+		TLSHandshakeTimeout:   defaultTransport.TLSHandshakeTimeout,
+		TLSClientConfig:       &tls.Config{InsecureSkipVerify: insecure},
+	}
```

Comment:
m.b. use compiler copying?
defaultTransport := *(http.DefaultTransport.(*http.Transport))
defaultTransport. TLSClientConfig = &tls.Config{InsecureSkipVerify: insecure}
return &http.Client{Transport: transport}

Reply:
I've just copied that from StackOverflow. =) Your version is better, thanks.